### ISSUE TITLE
Support for PHP 7.4

### ISSUE THREAD

**Issue by lgrassini** [F21/DNSMadeEasy#13]
Legacy signature  for PHP `implode` has been deprecated as of PHP 7.4.0: https://www.php.net/manual/en/function.implode.php

PHP function `implode` are expecting the first parameter to be the separator (string) and the second parameter to be an array.

Using this library with PHP 7.4 or newer it is causing the following error exception:

```
ErrorException: implode(): Passing glue string after array is deprecated. Swap the parameters in file /vendor/f21/dnsmadeeasy/DNSMadeEasy/resource/Records.php on line 109
```

**Comment by F21** [comment]
Thanks for reporting this! If you can open a PR to fix the instances of `implode()` being used, I am happy to review and merge it.

**Comment by lgrassini** [comment]
PR created: https://github.com/F21/DNSMadeEasy/pull/14

**Comment by F21** [comment]
Thanks! This has been published as `1.3.0`.
